### PR TITLE
Submissions

### PR DESCRIPTION
**Pull request by AhmedJamal93**

### CODE REVIEW

File: src/Components/Dashboard/Cards/JobPostingCard.js [hunk 1]
```diff
@@ -104,10 +199,54 @@ function withMyHook(Component) {
...
+    const submissions = await httpGet(
+      "job-applications?jobId=" + jobId,
+      (await Auth.currentSession()).getIdToken().getJwtToken()
+    );
+
+    this.setState({
+      open: true,
+      submissions: submissions.data,
+    });
+    console.log(this.state.submissions);
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Cards/JobPostingCard.js:223]: Console log!

File: src/Components/Dashboard/Submissions.js [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+  //       "jobs",
+  //       (await Auth.currentSession()).getIdToken().getJwtToken()
+  //     );
+  //     if (existingJobsData.data.jobs !== undefined) {
+  //       this.setState({
+  //         jobs: existingJobsData.data.jobs,
+  //         isJobAppsLoaded: true,
+  //       });
+  //     }
+  //   };
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Submissions.js:147]: Commented out code

File: src/Components/Dashboard/Submissions.js [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+    // const cutOff = this.props.isSeniorExec ? 5 : 3;
+    // if (jobsData.data.jobs !== undefined) {
+    //   this.setState({
+    //     isJobpostsLoaded: true,
+    //     job_postings:
+    //       jobsData.data.jobs.length > cutOff
+    //         ? jobsData.data.jobs.slice(0, cutOff)
+    //         : jobsData.data.jobs,
+    //   });
+    // }
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Submissions.js:175]: Log and commented out code

**Review by ahmedhamodi — COMMENTED** [review]
A few places need clean up, but rest looks good to me.

**Comment by arsidada** [comment]
@AhmedJamal93 
This needs some work

image

1. No icons are loading for me
2. We need to only show submissions and not the job description and stuff. Job description and everything should be a separate modal triggered by a separate button
3. Why is it squished all the way to the left? I think the Grid needs to be fixed

**Review by arsidada — APPROVED** [review]

**Pull request merged**